native/src/components/AnchorIcon.tsx

```diff
@@ -48,7 +48,7 @@ class AnchorIcon extends React.Component<PropsType> {
     }
   }
 
-  render() {
+  render(): ReactNode {
```

Comment:
Maybe we should add an exception for react components. The return type is always clear here.